lib/teracy-dev-essential/config/rsync_recovery.rb

```diff
@@ -7,6 +7,12 @@ module Config
     class RsyncRecovery < TeracyDev::Config::Configurator
 
       def configure_common(settings, config)
+
+        if invalid_argv?
+          @logger.warn("please use 'vagrant up' instead of 'vagrant gatling-rsync-auto', so that rsync can have the crash auto recovery.")
+          abort
```

Comment:
we need `vagrant up` for user, should not abort here

Reply:
@datphan If not abort, the program will not stop and run gatling-rsync. Also, this case occurs when vagrant has been up so abort is not affected. Abort will stop the command!